server/app/lib/actions/fusor/deployment/open_shift/install_ose.rb

```diff
@@ -1,3 +1,15 @@
+#
+# Copyright 2015 Red Hat, Inc.
```

Comment:
Yes will make this 2016 and squash commit once everything is tested and ready to be merged.